app/controllers/planned_runs_controller.rb

```diff
@@ -77,6 +100,10 @@ def set_group
   end  
 
   def planned_run_params
-    params.require(:planned_run).permit(:date, :group_id, :training_plan, :progress)
+    params.require(:planned_run).permit(:date, :group_id, :training_plan, :progress, :stuff)
```

Comment:
Totally forgot to remove!! Removed: https://github.com/o19s/thats-trackable/commit/74be4b42a59177ea19e5315a948fac532e7817e3